src/template-tags/tickets.php

```diff
@@ -1809,3 +1809,16 @@ function tribe_tickets_new_views_is_enabled() {
 		return (bool) apply_filters( 'tribe_tickets_new_views_is_enabled', $enabled );
 	}
 }
+
+if ( ! function_exists( 'tribe_tickets_get_provider_slug' ) ) {
+	/**
+	 * Returns the slug for providers
+	 *
+	 * @since TBD
+	 *
+	 * @return string String for which the slug should be named.
+	 */
+	function tribe_tickets_get_provider_slug() {
+		return Tribe__Tickets__Tickets::get_provider_url_slug();
+	}
+}
```

Comment:
We shouldn't use `function_exists` anymore for these new functions.
This name is a little too generic, lets try to name it specifically about the query_arg.
Suggestions:
- `tribe_tickets_get_provider_query_arg_key`
- `tribe_tickets_get_provider_query_key`
- `tribe_tickets_get_provider_query_arg_slug`
- `tribe_tickets_get_provider_query_slug`

Reply:
I vote for `tribe_tickets_get_provider_query_slug` 💯